### ISSUE TITLE
Cannot install v0.2.3

### ISSUE THREAD

**Issue by iorate** [tophat/semantic-release-firefox-add-on#93]
Thank you for merging my PR (#90), fixing it (sorry for not knowing that `webExt.cmd.sign` uses a temporary directory) and adding tests.

However, I cannot install v0.2.3 to [my project](https://github.com/iorate/uBlacklist) because the postinstall step fails:
```
> semantic-release-firefox-add-on@0.2.3 postinstall /path/to/my/project/node_modules/semantic-release-firefox-add-on
> yarn-deduplicate

yarn-deduplicate: command not found
npm WARN semantic-release-firefox-add-on@0.2.3 requires a peer of semantic-release@^16.0.0 but none is installed. You must install peer dependencies yourself.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! semantic-release-firefox-add-on@0.2.3 postinstall: `yarn-deduplicate`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the semantic-release-firefox-add-on@0.2.3 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
It's because `yarn-deduplicate` is a dev dependency so isn't installed to my project, and even if I install `yarn-deduplicate` manually, it fails because my project has no `yarn.lock` (using `npm` and `package-lock.json` instead).

Would you please update the postinstall script for non-yarn projects?

**Comment by iamogbz** [comment]
@iorate oops, missed that. Will fix and update.

**Comment by iamogbz** [comment]
Fixed in `v0.2.4`
